tide_landing_page.install

```diff
@@ -1365,6 +1365,9 @@ function tide_landing_page_update_8034() {
  * Updates cards.
  */
 function tide_landing_page_update_8035(&$sandbox) {
+  if (!getenv("TIDE_LANDING_PAGE_RUN_8035")) {
```

Comment:
Why not just do it on `LAGOON_ENVIRONMENT_TYPE`?

Reply:
thanks @anthony-malkoun . like 
```
if (getenv('LAGOON_ENVIRONMENT_TYPE') != 'producton'){
return;
}
```
?

Reply:
Except for the spelling on `production` yeah that should work :). What's to guarantee this won't break on production though?

Reply:
Hi @anthony-malkoun , changed.  Thanks.

Reply:
@anthony-malkoun if it doesn't break in prod-left, should not break in production. Isn't it! That is my assumption :/